src/index.mjs

```diff
@@ -42,14 +42,14 @@ function prefetcher(url) {
  * @param {Object} options - Configuration options for quicklink
  * @param {Array} options.urls - Array of URLs to prefetch (override)
  * @param {Object} options.el - DOM element to prefetch in-viewport links of
- * @param {string} options.priority - Attempt to fetch with higher priority (low or high)
+ * @param {Boolean} options.priority - Attempt higher priority fetch (low or high)
```

Comment:
I initially planned for priority to have a tri (or more) state set of values, but it's very likely to stay at either high or low. With that in mind, strongly in favor of switching to a `Boolean` here 👍

Reply:
Ah, gotcha. I didn't even know there _could_ be other priorities 😆